relay-quotas/src/is_rate_limited.lua

```diff
@@ -50,7 +51,7 @@ for i=0, num_quotas - 1 do
         -- With over_accept_once, we only reject if the previous update already reached the limit. 
         -- This way, we ensure that we increment to or past the limit at some point,
         -- such that subsequent checks with quantity=0 are actually rejected.
-        if quantity == 0 or over_accept_once == 'true' then
```

Comment:
Why is this changing?

Reply:
Is this the only place where we pass parameters into a Lua script?

Reply:
As far as I see, all the invocations are done in `relay-quotas/src/redis.rs`  where we prepare and pass all the parameters to this script

Reply:
I think it's beneficial to add a comment to note that.

Reply:
@iker-barriocanal  added a note in [4d0ca6b](https://github.com/getsentry/relay/pull/1538/commits/4d0ca6b2d8d586d2cbad897f9f5471f80043ca7f)